src/snapred/backend/service/ServiceFactory.py

```diff
@@ -11,7 +11,8 @@
 from snapred.backend.service.CrystallographicInfoService import CrystallographicInfoService
 from snapred.backend.service.LiteDataService import LiteDataService
 from snapred.backend.service.NormalizationService import NormalizationService
-from snapred.backend.service.ReductionService import ReductionService
+
+# from snapred.backend.service.ReductionService import ReductionService
```

Comment:
I would either remove this entirely, or add a TODO if you plan to add this back later, same for below